service/command_CgmQuit.go

```diff
@@ -0,0 +1,18 @@
+package service
+
+import (
+	"time"
+
+	"github.com/blockassets/cgminer_client"
+)
+
+const (
+	MINER_HOSTNAME = "localhost"
+	MINER_PORT     = 4028
+	MINER_TIMEOUT  = 5 * time.Second
+)
+
+func (*Command) CgmQuit() error {
+	clnt := cgminer_client.New(MINER_HOSTNAME, MINER_PORT, MINER_TIMEOUT)
```

Comment:
I'd put this in the initial startup of the agent and then call it as necessary. Likely we will want to be able to pass these options in during the agent startup, but that can be added later.